testcases/PowerNVDump.py

```diff
@@ -605,7 +605,6 @@ def setup_fadump(self):
         self.cv_SYSTEM.set_state(OpSystemState.OS)
         if self.distro == "rhel":
             self.c.run_command("sed -e '/nfs/ s/^#*/#/' -i /etc/kdump.conf; sync")
-            self.c.run_command("sed -i 's/path \//path \/var\/crash/' /etc/kdump.conf; sync")
```

Comment:
Any reason to remove this line ?

Reply:
This line is not required as now we are replacing the config file with original file after each test. and this replacement may change the path variable to undesired value.